cfme/utils/ssh.py

```diff
@@ -527,10 +527,7 @@ def status(self):
                 'Please use .* instead',
                 'key :terminate is duplicated and overwritten',
             ]))
-        if version.current_version() < "5.5":
-            data = self.run_command("systemctl status evmserverd")
-        else:
-            data = self.run_rake_command("evm:status")
+        data = self.run_rake_command("evm:status")
```

Comment:
I believe this whole method should exist under `IPAppliance` object, not here, but that's (understandably) out of scope for this PR.